### PR TITLE
Update tracker configuration with a new cookie name for the biz1 tracker

### PR DESCRIPTION
**Pull request by matus-tomlein**
Updates the configuration for `snplow5` and `biz1` trackers:

* sets the method to POST on both trackers
* sets `cookieSameSite` to `Lax` on both trackers
* changes the `cookieName` for `biz1` tracker

### CODE REVIEW

**Review by igneel64 — APPROVED** [review]
LGTM

File: snowplow.js [hunk 1]
```diff
@@ -15,17 +15,19 @@ import Cookies from 'js-cookie'
...
 
-const setupBrowserTracker = () => {
+const createTrackerConfig = (cookieName) => {
   const appId = DOCS_SITE_URLS.includes(window.location.hostname)
     ? 'docs2'
     : 'test'
   const domain = location.host.split('.').reverse()
 
-  const trackerConfig = {
+  let trackerConfig = {
```
Comment (stanch) [inline snowplow.js:24]: why?

File: snowplow.js [hunk 2]
```diff
@@ -41,8 +43,12 @@ const setupBrowserTracker = () => {
...
   }
 
-  newTracker('snplow5', 'https://collector.snowplow.io', trackerConfig)
-  newTracker('biz1', 'https://c.snowplow.io', trackerConfig)
+  return trackerConfig;
+};
+
+const setupBrowserTracker = () => {
+  newTracker('snplow5', 'https://collector.snowplow.io', createTrackerConfig('_sp5_'))
+  newTracker('biz1', 'https://c.snowplow.io', createTrackerConfig('_sp_biz1_'))
```
Comment (stanch) [inline snowplow.js:51]: I think we should add the new cookie here: https://github.com/snowplow/documentation/blob/main/src/pages/cookie-preferences.mdx?plain=1#L64

**Review by stanch — CHANGES_REQUESTED** [review]

File: snowplow.js (hunk 1 above)
Comment (matus-tomlein) [reply snowplow.js]: I misunderstood what the const keyword means. I thought it meant that the object is a constant and can't be changed (which it is on line 41 below). But reading more about it I see that it only prevents re-assigning and redeclaration. Good to know!

File: snowplow.js (hunk 2 above)
Comment (matus-tomlein) [reply snowplow.js:51]: Good point, done!

**Review by stanch — APPROVED** [review]

**Pull request merged**